datasets/flwr_datasets/visualization/comparison_label_distribution.py

```diff
@@ -132,6 +133,13 @@ def plot_comparison_label_distribution(
     >>>     titles=[f"Concentration = {alpha}" for alpha in alpha_list],
     >>> )
     """
+    event(
+        EventType.PLOT_COMPARISON_LABEL_DISTRIBUTION_CALLED,
+        {
+            "num_compare": len(partitioner_list),
```

Comment:
Track `len(partitioner_list)` bc this fnc uses internally the `plot_label_distribution` (so we will know how many events were created bc this function was used and adjust the metrics).